### DISCUSSION TITLE
Ability to Suppress more Warnings

### DISCUSSION THREAD

**Discussion by gmusgrave** [pymedusa/Medusa#7550] [Ideas & Feature requests]
**Is your feature request related to a problem? Please describe.**
I frequently get warnings about providers such as Rarbg and ozNZB in the form of "_Unable to connect to provider_" or "_Error getting caps xml for: xxx_."

These appear to be transient issues that don't affect the overall performance or functionality of Medusa, but I still have to clear these warnings – often, several times a week. 

In addition to their having no perceptible effect on my user experience, there is nothing I can do about them anyway. I'd really rather just not have to deal with them.

Not an earth-shaking problem, but annoying.

**Describe the solution you'd like**
I'd like a config option with checkboxes to disable various classes of warnings that I don't need to see, and don't want to have to take action about.

The warnings should still appear in the logs, I just don't want to see the alert, and have to clear it.

**Comment by zatricky** [comment]
Possibly the best way to address this issue is to move away from `alert()` to the vue-snotify already being used for other notifications. I believe they already show up in the Error log at least. The subsequent option is to allow sending these "priority" notifications to the "3rd-party" notification channels.
Currently, where `alert()` is used, we have eslint configs specifically to ignore the fact that it is in use. See https://eslint.org/docs/rules/no-alert for more on why using them is discouraged.

**Comment by p0psicles** [comment]
We don't use alert()

**Comment by zatricky** [comment]
My apologies ; I'm actually seeing a different issue in the themes' code (where you will find `alert()` sprinkled about). I'll make a separate issue.
